thirdparty/imgui_suite/build.gradle

```diff
@@ -24,7 +24,6 @@ nativeUtils.platformConfigs.named('osxuniversal') {
 }
 
 model {
-
```

Comment:
Unrelated change?

Reply:
Yeah, but I figured I may as well remove it.